guides/_subsections/archive-a-recording.md

```diff
@@ -21,14 +21,14 @@ any state (`RUNNING`, `STOPPED`, etc.), and may even be a snapshot.
   <li>
     {% capture select-recording-additional-content %}
       <p>
-        If you do not have any recordings present in the Active Recordings
+        If you do not have any <code>Recordings</code> present in the <i>Active Recordings</i>
         view, follow
-        <a href="{{ page.url }}#startstop-a-recording">Start/Stop a Recording</a>
+        <a href="{{ page.url }}#startstop-a-recording"><i>Start/Stop</i> a <code>Recording</code></a>
```

Comment:
I think because this is a link to another section, you can just leave it as "Start/Stop a Recording"? The link's underline is disjointed because of the code block 
![Screenshot from 2023-11-24 11-49-41](https://github.com/cryostatio/cryostatio.github.io/assets/90855268/96c3e5bc-0578-46ca-bcd2-a9facfb5a2f7)